ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h

```diff
@@ -426,7 +426,7 @@ std::vector<double> const& ThermoHydroMechanicsLocalAssembler<
         // Compute the velocity
         auto const& dNdx_p = _ip_data[ip].dNdx_p;
         cache_matrix.col(ip).noalias() =
-            -K_over_mu * dNdx_p * p + K_over_mu * rho_fr * b;
```

Comment:
This change is not related to this PR.

Reply:
Please rebase. @HBShaoUFZ

Reply:
Will vanish after a rebase.
My bad. Had to revert some changes yesterday.